samples/server/petstore/jaxrs-resteasy/default/src/gen/java/org/openapitools/model/Category.java

```diff
@@ -21,6 +23,7 @@
   
   @ApiModelProperty(value = "")
   @JsonProperty("id")
+  
```

Comment:
Which code style you following to put whitespace between method and annotations? Isn't redundant?